### PR TITLE
LPS-177877 - Automation Forms | Master

### PR DESCRIPTION
**Pull request by thaiscabrals**
Bug: [LPS-177877](https://liferay.atlassian.net/browse/LPS-177877)
Automation:[LPS-194182](https://liferay.atlassian.net/browse/LPS-194182)
Test: ViewWidgetPageAtTheTop

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Hey @thaiscabrals, based from our conversation earlier, we want to make sure that this test actually covers the bug you are automating. Thank you!

**Comment by thaiscabrals** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 1]
```diff
@@ -2379,6 +2379,80 @@ definition {
...
+			FormPortletNavigator.gotoSelectForm();
+
+			FormPortletConfiguration.selectForm(formName = "Form 1");
+
+			PortletEntry.save();
+
+			IFrame.closeFrame();
+		}
+
+		task ("e When the user leaves the page, returns to the page and refreshes") {
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:2441]: @thaiscabrals typo here

**Comment by thaiscabrals** [comment]
I'm working on it

**Comment by thaiscabrals** [comment]
ci:test:sf

**Pull request closed without merging**